ckanext/ontario_theme/templates/internal/package/resource_edit_base.html

```diff
@@ -18,11 +18,31 @@
         {% block page_header %}
         {% endblock page_header %}
         <div class="module-content">
-          <h1>
-            {% block form_title %}
-              {{ super() }}
-            {% endblock form_title %}
-          </h1>
+          <div class="ontario-row">
+            <div class="ontario-small-7 ontario-columns">
+              <h1>
+                {% block form_title %}
+                  {{ super() }}
+                {% endblock form_title %}
+              </h1>
+            </div>
+            <div id="delbtn" class="ontario-small-5 ontario-columns">
+              {% block delete_button %}
+                {% if data %}
+                  {% if data.id %}
+                    {% if h.check_access('resource_delete', {'id': data.id}) %}
+                      <a class="btn btn-danger pull-right ontario-button ontario-button--secondary"
```

Comment:
In the css, targeting `#delbtn a`, set the margin-right to 0 to make the button aligned to the right side of the page.